.github/workflows/ci.yml

```diff
@@ -20,6 +20,6 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: ">=14.16.0 14"
+           node-version: ">=16.16.0 16"
```

Comment:
Let's change the `npm test` command to run on both node 14 and node 16 (The current active and maintenance lts versions).

![image](https://user-images.githubusercontent.com/2328042/183615319-000a196e-638d-470e-9653-fb9ee199862b.png)